Tests/DatadogTests/Datadog/RUM/RUMEvent/RUMEventSanitizerTests.swift

```diff
@@ -141,18 +141,18 @@ class RUMEventSanitizerTests: XCTestCase {
             let sanitized = RUMEventSanitizer().sanitize(event: event)
 
             // Then
-            XCTAssertEqual(
-                sanitized.attributes.count + sanitized.userInfoAttributes.count + (sanitized.customViewTimings?.count ?? 0),
-                AttributesSanitizer.Constraints.maxNumberOfAttributes
-            )
-            XCTAssertTrue(
-                (sanitized.customViewTimings?.count ?? 0) >= sanitized.userInfoAttributes.count,
-                "If number of attributes needs to be limited, `userInfoAttributes` are removed prior to `customViewTimings`."
-            )
-            XCTAssertTrue(
-                sanitized.userInfoAttributes.count >= sanitized.attributes.count,
-                "If number of attributes needs to be limited, `attributes` are removed prior to `userInfoAttributes`."
-            )
+            var remaining = AttributesSanitizer.Constraints.maxNumberOfAttributes
+            let expectedSanitizedCustomTimings = min(sanitized.customViewTimings!.count, remaining)
+            remaining -= expectedSanitizedCustomTimings
+            let expectedSanitizedUserInfo = min(sanitized.userInfoAttributes.count, remaining)
+            remaining -= expectedSanitizedUserInfo
+            let expectedSanitizedAttrs = min(sanitized.attributes.count, remaining)
+            remaining -= expectedSanitizedAttrs
+
+            XCTAssertGreaterThanOrEqual(remaining, 0)
+            XCTAssertEqual(sanitized.customViewTimings!.count, expectedSanitizedCustomTimings, "If number of attributes needs to be limited, `customViewTimings` are removed last")
```

Comment:
Nitpick: it's not a big deal but to help following along, maybe reverse the order of the asserts? Like so:
- If number of attributes needs to be limited, `attributes` are removed first.
- If number of attributes needs to be limited, `userInfoAttributes` are removed second
- If number of attributes needs to be limited, `customViewTimings` are removed last

Reply:
i followed the same order above while computing expected values, this is more readable for me personally.
i'm merging as it is now, okay?

Reply:
That sounds good 👍